### PR TITLE
vtol_att_control vtol_type enum -> enum class

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: src/modules/vtol_att_control/tailsitter.h [hunk 1]
```diff
@@ -43,7 +43,7 @@
 #define TAILSITTER_H
 
 #include "vtol_type.h"
-#include <perf/perf_counter.h>  /** is it necsacery? **/
+
```
Comment (julianoes) [inline src/modules/vtol_att_control/tailsitter.h]: :smile:

File: src/modules/vtol_att_control/tailsitter.h [hunk 2]
```diff
@@ -73,7 +73,7 @@ class Tailsitter : public VtolType
 		param_t fw_pitch_sp_offset;
 	} _params_handles_tailsitter{};
 
-	enum vtol_mode {
+	enum class vtol_mode {
```
Comment (julianoes) [inline src/modules/vtol_att_control/tailsitter.h]: Did I miss this one?

File: src/modules/vtol_att_control/tiltrotor.h [hunk 3]
```diff
@@ -75,7 +75,7 @@ class Tiltrotor : public VtolType
 		param_t front_trans_dur_p2;
 	} _params_handles_tiltrotor;
 
-	enum vtol_mode {
+	enum class vtol_mode {
```
Comment (julianoes) [inline src/modules/vtol_att_control/tiltrotor.h]: And this one too? So this same enum is declared everywhere?

**Comment by LorenzMeier** [comment]
It all seems to make sense to me, but is the conversion of the standard VTOL maybe missing?

**Comment by dagar** [comment]
Each vtol type has a private vtol_mode enum (now enum class). In standard it was already enum class.

File: src/modules/vtol_att_control/tiltrotor.h (hunk 3 above)
Comment (dagar) [reply src/modules/vtol_att_control/tiltrotor.h]: Per type privately.

**Pull request merged**